src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -197,232 +182,21 @@ $border-radius: 28px;
   padding: 10px;
 }
 
-.room {
```

Comment:
Note: The relevant room/etc styles from here were moved into `src/components/templates/PartyMap/components/Map/MapRoom.scss`